src/server_soc_requirements.adoc

```diff
@@ -1266,6 +1266,24 @@ data centers and enterprises.
 [%header, cols="5,25"]
 |===
 | ID#     ^| Requirement
+| SEC_005 a| The Server SoC MUST comply with the requirements detailed in
+             Reference Model section and Ecosystem Security Objectives section
+             of the RISC-V Security Model cite:[SEC]. The Server SoC is
+             classified as a complex security system for the purposes of
+             SR_ROT_001 and SR_ATT_002.
+
+| SEC_006 a| The Server SoC MUST support the Generic System Without Supervisor
+             Domains use case detailed in the RISC-V Security Model. The
+             building blocks used to implement this use case MUST comply with
+             the requirements specified in the RISC-V Security Building Blocks
+             section of the RISC-V Security Model specification.
```

Comment:
should say something about section 4.2 related requirements - SR_GEN_007..016 are included I presume